src/DssDirectDepositHub.t.sol

```diff
@@ -362,13 +362,13 @@ contract DssDirectDepositHubTest is DSTest {
     function test_wind_limited_maxDeposit() public {
```

Comment:
Can we add an extra test where the `maxDeposit` amount is something bigger than 50 but lower than 75, to show that wind still will do something restricted to the `maxDeposit`?